instance/models/openedx_appserver.py

```diff
@@ -362,18 +337,29 @@ def smtp_relay_settings(self):
             return None
 
     @property
-    def github_admin_username_list(self):
+    def admin_users(self):
         """
-        Returns the github usernames of this instance admins
+        Return GitHub usernames of admin users for this instance.
 
-        Admins are all users listed in github_admin_users and the members of the default team of the
-        organizations in github_admin_organizations.
+        Admins are all users with `is_superuser` set to `True`,
+        as well as all members of the organization that owns this instance.
         """
-        admin_users = []
-        for org in self.github_admin_organizations:
-            admin_users += get_username_list_from_team(org)
-        admin_users += self.github_admin_users
-        return admin_users
+        users = []
+        # Add members of owning organization to the list of users that can administer this instance
+        organization = self.instance.ref.owner
+        if organization:
+            usernames = UserProfile.objects.filter(
+                organization=organization
+            ).values_list('github_username', flat=True)
+            users += usernames if usernames else []
+
+        # Add superusers to the list of users that can administer this instance
+        usernames = UserProfile.objects.filter(
+            user__is_superuser=True
+        ).values_list('github_username', flat=True)
+        users += usernames if usernames else []
+
+        return users
```

Comment:
I tested this and found that it also adds a username twice if it's superuser and belongs to an Organization. You may want to use list(set(…)) or something similar at the end so that each username appears once. I don't know the effects of being there twice, but maybe SSH later can complain.